pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/NotificationUtil.java

```diff
@@ -79,8 +79,9 @@ public static Long getLastChanged(@NonNull String participantId) {
 
     public synchronized void startup(@NonNull Config config) {
         emailKey = config.getString(ApplicationConfigConstants.EMAIL_KEY);
-        emailProxyUrl = config.getString(ApplicationConfigConstants.EMAIL_PROXY_URL);
-
+        if (config.hasPath(ApplicationConfigConstants.EMAIL_PROXY_URL)) {
+            emailProxyUrl = config.getString(ApplicationConfigConstants.EMAIL_PROXY_URL);
```

Comment:
What happens elsewhere in the code if `emailProxyUrl` is not set?

Reply:
Nothing--the part of the code that responds to its null-ness is unchanged.  See `SendGridFactory` for details.